soc/arm/nordic_nrf/Kconfig.defconfig

```diff
@@ -28,9 +28,6 @@ endif # GPIO
 
 if FLASH
 
-config SOC_FLASH_NRF
```

Comment:
Why did you remove this?

Reply:
Because kconfig will barf out warnings and whatnot about this now that it is a (sub-)choice of drivers. I could remove the whole `if FLASH ... endif` structure from here.